src/utils/sortTableValues.js

```diff
@@ -0,0 +1,20 @@
+const sortTableValues = (key, order = "asc") => {
+  return (a, b) => {
+    if (!a.hasOwnProperty(key) || !b.hasOwnProperty(key)) {
+      return 0;
+    }
+
+    const varA = typeof a[key] === "string" ? a[key].toUpperCase() : a[key];
+    const varB = typeof b[key] === "string" ? b[key].toUpperCase() : b[key];
+
+    let comparison = 0;
+    if (varA > varB) {
+      comparison = 1;
+    } else if (varA < varB) {
+      comparison = -1;
+    }
+    return order === "desc" ? comparison * -1 : comparison;
+  };
+};
+
+export default sortTableValues;
```

Comment:
In utils files is not common to have a default export. It might have multiple function utilities and each utility should be exported as a named export.

Reply:
Like `export const sortTableValue....`

Reply:
updated